src/Lint.jl

```diff
@@ -263,33 +275,40 @@ end
 
 function lintserver(port)
     server = listen(port)
+    println("Server running on port $port ...")
     while true
-      conn = accept(server)
-      @async begin
-        try
-          while true
-            line = readline(conn)
-
-            println(typeof(line))
-            println(ispath(strip(line)))
-            println(strip(line))
-
-            if ispath(strip(line))
-                m = lintfile(strip(line), returnMsgs = true)
-            else
-                error("The string is not a file.")
+        conn = accept(server)
+        @async try
+            println("Connection accepted.")
+            # Get file, code length and code
+            file = strip(readline(conn))
+            println("file: ", file)
+            code_len = parse(Int, strip(readline(conn)))
+            println("Code bytes: ", code_len)
+            code = utf8(readbytes(conn, code_len))
+            println("Code received")
+            # Build context
+            ctx = LintContext()
+            ctx.file = file
+            if ispath( file )
+                ctx.path = dirname( file )
             end
-
-            for i in m
+            # Lint code
+            msgs = lintstr( code, ctx )
+            # Process messages
+            clean_messages!( msgs )
+            display_messages( msgs )
```

Comment:
There's allot of mixed styling going on. Try stick to `func( arg )` or `func(arg)` within a single function